specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/machineLearningServices.json

```diff
@@ -1273,6 +1273,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/updateIdleShutdownSetting": {
+      "post": {
+        "tags": [
+          "MachineLearningComputes"
+        ],
+        "operationId": "Compute_UpdateIdleShutdownSetting",
```

Comment:
>Compute_UpdateIdleShutdownSetting [](http://example.com/codeflow?start=24&length=33)

Why isn't this modeled as e.g. PATCH on a compute resource instance?

Reply:
This is how it was designed since the outset. However we now understand that this is not the ideal way to update compute settings and we have plan in place to support update on compute through PUT/PATCH. However this can take time as we do it for each feature/setting. This update setting API is a stop gap measure till we can fully support the PATCH scenario.

Reply:
This API looks like it is just going to become technical debt if there was no specific reason for adding it.

Reply:
removed the POST api to update the setting separately.